frontends/open-discussions/src/pages/CourseSearchPage_test.js

```diff
@@ -394,7 +394,7 @@ describe("CourseSearchPage", () => {
     wrapper.find(".clear-all-filters").simulate("click")
     await wait(1)
     const search = pushStub.args[pushStub.args.length - 1][2]
-    assert.deepEqual(search, "/?")
+    assert.deepEqual(search, "/")
```

Comment:
This is the *only* change to any of the application / application test files.

I'm guessing a minor bump in one of our dependencies changed how it deals with empty querystrings. The new behavior seems better anyway.